src/main/java/uk/gov/companieshouse/model/dto/DissolutionGetResponse.java

```diff
@@ -0,0 +1,120 @@
+package uk.gov.companieshouse.model.dto;
+
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import uk.gov.companieshouse.model.db.DissolutionGetDirector;
+import uk.gov.companieshouse.model.enums.ApplicationStatusEnum;
+import uk.gov.companieshouse.model.enums.ApplicationTypeEnum;
+
+import java.sql.Timestamp;
+import java.util.List;
+
+public class DissolutionGetResponse {
+
+    private String ETag;
+    private String kind;
+    private DissolutionLinks links;
+    @JsonProperty("application_status")
+    private ApplicationStatusEnum applicationStatus;
```

Comment:
I really dislike suffixing models with their type. This should just be ApplicationStatus instead of ApplicationStatusEnum